### PR TITLE
Homework3: Ivan Veremeichik

### PR DESCRIPTION
**Pull request by NorgeMan**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework3/task1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+list_ = [number for number in range(1, 101)]
+final_list = []
+
+for number in list_:
+    if not (number % 3):
+        final_list.append("Fizz")
+    elif not (number % 5):
+        final_list.append("Buzz")
+    elif not (number % 3) and not (number % 5):
```
Comment (belovmd) [inline src/homework3/task1.py]: 15?

File: src/homework3/task2.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# subtask 1
+list_1 = [letter_1 + letter_2 for letter_1 in "ab" for letter_2 in "bcd"]
+print(list_1)
+
+# subtask 2
+print(list_1[::2])
+
+# subtask 3
+list_2 = [str(number) + "a" for number in range(1, 5)]
```
Comment (belovmd) [inline src/homework3/task2.py]: '1234'

File: src/homework3/task4.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+# Input: 1 1 1 1 9 9 9 10 10 12 12 12 14 14 14 14 15
+# Output: 19
+
+list_ = [int(number) for number in input("Enter the elements: ").split()]
+counter = 0
+
+for element in list_:
+    if list_.count(element) > 1:
+        counter += (list_.count(element) - 1)
```
Comment (belovmd) [inline src/homework3/task4.py]: arithmetical progression

**Review by belovmd — CHANGES_REQUESTED** [review]

File: src/homework3/task1.py (hunk 1 above)
Comment (IVeremeichik) [reply src/homework3/task1.py]: Done

File: src/homework3/task2.py (hunk 2 above)
Comment (IVeremeichik) [reply src/homework3/task2.py]: Done

File: src/homework3/task4.py (hunk 3 above)
Comment (IVeremeichik) [reply src/homework3/task4.py]: Done

File: src/homework3/task1.py (hunk 1 above)
Comment (NorgeMan) [reply src/homework3/task1.py]: This phrases were left from my second account

File: src/homework3/task2.py (hunk 2 above)
Comment (NorgeMan) [reply src/homework3/task2.py]: This phrases were left from my second account

File: src/homework3/task4.py (hunk 3 above)
Comment (NorgeMan) [reply src/homework3/task4.py]: This phrases were left from my second account

**Comment by belovmd** [comment]
9

**Pull request closed without merging**